cinema/views.py

```diff
@@ -103,7 +138,7 @@ def get_queryset(self):
 
         return queryset
 
-    def get_serializer_class(self):
+    def get_serializer_class(self) -> type[serializers.Serializer]:
```

Comment:
```suggestion
    def get_serializer_class(self) -> Type[serializers.Serializer]:
```

`type` is a function

Reply:
Fix everywhere